### PR TITLE
Add eligibility criteria to datasets.toml

### PR DESCRIPTION
**Pull request by J535D165**
@Rensvandeschoot @EmilyWes extracted the eligibility criteria from the supplementing publication for each synergy dataset. This PR adds the quoted sections to the TOML metadata for the SYNERGY datasets. 

These sections are especially relevant to machine learning applications based on chatbot/LLM technology. For more use cases, please take a look at issue #94. 

In another PR, I will update the release pipeline and add the criteria to the released dataset (however, feel free to use it from the TOML file already).

### CODE REVIEW

File: datasets.toml [hunk 1]
```diff
@@ -1,635 +1,670 @@
...
+The search was not limited by the year of publication. Journal papers and conference proceedings were included. The search was limited to English.
 
-  [datasets.collection]
-  doi="10.1007/s10664-017-9587-0"
+During the systematic review, we included empirical studies validating and assessing software metrics in the area of software fault prediction. For the selection of primary studies, the following inclusion and exclusion criteria were used.
 
-  [datasets.publication]
-  doi="10.1016/j.infsof.2010.03.006"
+Inclusion criteria:
+_x0002_- Empirical studies (academic and industry) using large and small scale data sets AND.
```
Comment (J535D165) [inline datasets.toml:58]: hmmm. I think I will fix this by hand.

File: datasets.toml [hunk 2]
```diff
@@ -1,635 +1,670 @@
...
+_x0002_- Studies comparing software metrics performance in the area of software fault prediction.
```
Comment (J535D165) [inline datasets.toml:59]:
```suggestion
• Empirical studies (academic and industry) using large and small scale data sets AND.
• Studies comparing software metrics performance in the area of software fault prediction.
```

**Review by EmilyWes — CHANGES_REQUESTED** [review]
I think all criteria should start and end with a quote, resulting in triple quotes in the file. This to make clear they are blocked quotes from in the papers.
Currently this is not yet the case for: Appenzeller-Herzog, Bos, Brouwer, Donners, Leenaars_2020, Moran, Oud, Sep and Wassenaar.

2 criteria are missing (their names changed over time):
van_de_Schoot and (van_der_)Valk

**Comment by J535D165** [comment]
Good catch on Valk and Schoot dataset. Will open another PR.

File: README.md [hunk 3]
```diff
@@ -24,7 +24,9 @@ To get an overview of the datasets and their properties, use `synergy_dataset li
 
 ## Datasets and variables
 
-The SYNERGY dataset comprises the study selection of 26 systematic reviews. The dataset contains 169,288 records of which 2,834 records are manually labeled as inclusion by the authors of the systematic review. The list of systematic review and basic properties:
+The SYNERGY dataset comprises the study selection of 26 systematic reviews. The dataset contains 169,288 records of which 2,834 records are manually labeled as inclusion by the authors of the systematic review. The eligibility criteria are added to the data as blocked quotes.
```
Comment (J535D165) [inline README.md:27]:
```suggestion
The SYNERGY dataset comprises the study selection of 26 systematic reviews. The dataset contains 169,288 records of which 2,834 records are manually labeled as inclusion by the authors of the systematic review. The eligibility criteria are added to the data as block quotations. 
```

**Comment by J535D165** [comment]
Thanks for the reviews. Helped.

@EmilyWes, I didn't incorporate your first suggestion because of the TOML specification https://toml.io/en/v1.0.0#string.

**Pull request merged**